PWGHF/vertexingHF/AliAnalysisTaskSEXic0pp13TeV.cxx

```diff
@@ -0,0 +1,1514 @@
+**************************************************************************/
+
+// Comment describing what this class does needed!
+//==================================================================
+// Xi_c^0 analysis
+// by Jinjoo Seo
+//==================================================================
+
+#include <TSystem.h>
+#include <TParticle.h>
+#include <TParticlePDG.h>
+#include <TH1F.h>
+#include <TH2F.h>
+#include <THnSparse.h>
+#include <TLorentzVector.h>
+#include <TTree.h>
+#include "TROOT.h"
+#include <TDatabasePDG.h>
+#include <AliAnalysisDataSlot.h>
+#include <AliAnalysisDataContainer.h>
+#include "AliMCEvent.h"
+#include "AliAnalysisManager.h"
+#include "AliAODMCHeader.h"
+#include "AliAODHandler.h"
+#include "AliExternalTrackParam.h"
+#include "AliAODVertex.h"
+#include "AliESDVertex.h"
+#include "AliRDHFCuts.h"
+#include "AliAODRecoDecay.h"
+#include "AliAODRecoDecayHF.h"
+#include "AliAODRecoCascadeHF.h"
+#include "AliESDtrack.h"
+#include "AliAODTrack.h"
+#include "AliAODcascade.h"
+#include "AliAODMCParticle.h"
+#include "AliAnalysisTaskSE.h"
+#include "AliAnalysisTaskSEXic0pp13TeV.h"
+#include "AliPIDResponse.h"
+#include "AliPIDCombined.h"
+#include "AliTOFPIDResponse.h"
+#include "AliAODPidHF.h"
+#include "AliInputEventHandler.h"
+#include "AliPPVsMultUtils.h"
+#include "AliESDtrackCuts.h"
+#include "AliESDtrack.h"
+#include "AliAODInputHandler.h"
+#include "AliAnalysisTaskSEHFTreeCreator.h"
+#include "AliMultSelection.h"
+
+
+const Double_t pi = TMath::Pi();
+enum {  kPN=1, kPP, kNN, kMixing, kAllType}; //P=Positive charge, N=Negative
+
+AliAnalysisTaskSEXic0RunTable::AliAnalysisTaskSEXic0RunTable() :
+  fCollisionType(kUnknownCollType){;}   //defult constructor
+
+AliAnalysisTaskSEXic0RunTable::AliAnalysisTaskSEXic0RunTable(Int_t fRunNumber){
+  //if (fRunNumber>=256504 && fRunNumber<=260014) fCollisionType=kPP; //LHC16kl
+  //else if (fRunNumber>=254604 && fRunNumber<=255467) fCollisionType=kPP; //LHC16h
+  fCollisionType=kPP;
+}
+AliAnalysisTaskSEXic0RunTable::~AliAnalysisTaskSEXic0RunTable() {;}
+
+//___________________________________________________________________
+AliAnalysisTaskSEXic0pp13TeV::AliAnalysisTaskSEXic0pp13TeV() :
+  AliAnalysisTaskSE("AliAnalysisTaskSEXic0pp13TeV"),
+  fOption(){
+    DefineOutput (1, TDirectory::Class());
+    DefineOutput (2, TList::Class());
+    DefineOutput (3, TTree::Class());
+    DefineOutput (4, TTree::Class());
+    DefineOutput (5, TTree::Class());
+    DefineOutput (6, TTree::Class());
+    DefineOutput (7, AliNormalizationCounter::Class());
+    fMCCutTreeVariable = 0;
+    fPaireXiTreeVariable = 0;
+    fMCTreeVariable = 0;
+    fEventTreeVariable = 0;
+    fMCCutTree = 0;
+    fPaireXiTree = 0;
+    fMCTree = 0;
+    fEventTree = 0;
+}
+//___________________________________________________________________
+AliAnalysisTaskSEXic0pp13TeV::AliAnalysisTaskSEXic0pp13TeV( const char *name, const char *option ) :
```

Comment:
I guess that this task can be used also in other systems, why not just use the name `AliAnalysisTaskSEXic0Semileptonic` or something like this?

Reply:
Thanks for your kindness comments. I revised all of comments and change the name as you mentioned.